src/context/playlist-context.js

```diff
@@ -1,28 +1,30 @@
 import React, { useState, useContext, createContext, useReducer, useEffect } from "react"
 import { playListReducer, playListTitleReducer } from "reducer";
-import { getPlaylistData } from "utilities"
+import { getPlaylistData, getWatchLaterData } from "utilities"
 
 const PlayListContext = createContext();
 
 const PlayListProvider = ({ children }) => {
 
     const [showModal, setShowModal] = useState(false);
     const [playListState, playListDispatch] = useReducer(playListReducer, {
-        playlists: []
+        playlists: [],
+        watchLater: []
     })
     const [playListTitleState, playListTitleDispatch] = useReducer(playListTitleReducer, {
         allPlayListTitle: [],
         playListTitle: []
     })
-    const { playlists } = playListState
+    const { playlists, watchLater } = playListState
     const { playListTitle, allPlayListTitle } = playListTitleState
 
     useEffect(() => {
         getPlaylistData(playListDispatch);
+        getWatchLaterData(playListDispatch);
```

Comment:
no need to fire this, as this is mockbee, although great that your code is capable for future adaptation